client/my-sites/checkout/concierge-session-nudge/concierge-support-session/index.jsx

```diff
@@ -0,0 +1,221 @@
+										'{{b}}Traffic:{{/b}} How to make your site search-engine friendly.',
+										{
+											components: { b: <b /> },
+											comment: "This is a benefit listed on a 'Purchase a call with us' page",
+										}
+									) }
+								</span>
+							</li>
+							<li className="concierge-support-session__checklist-item">
+								<Gridicon
+									icon="checkmark"
+									className="concierge-support-session__checklist-item-icon"
+								/>
+								<span className="concierge-support-session__checklist-item-text">
+									{ translate(
+										"{{b}}Site building tools:{{/b}} Learn how to create a site you're proud to share.",
+										{
+											components: { b: <b /> },
+											comment: "This is a benefit listed on a 'Purchase a call with us' page",
+										}
+									) }
+								</span>
+							</li>
+							<li className="concierge-support-session__checklist-item">
+								<Gridicon
+									icon="checkmark"
+									className="concierge-support-session__checklist-item-icon"
+								/>
+								<span className="concierge-support-session__checklist-item-text">
+									{ translate(
+										'{{b}}Content:{{/b}} What information to include and where it should go.',
+										{
+											components: { b: <b /> },
+											comment: "This is a benefit listed on a 'Purchase a call with us' page",
+										}
+									) }
+								</span>
+							</li>
+							<li className="concierge-support-session__checklist-item">
+								<Gridicon
+									icon="checkmark"
+									className="concierge-support-session__checklist-item-icon"
+								/>
+								<span className="concierge-support-session__checklist-item-text">
+									{ translate( "{{b}}And more:{{/b}} Tell our experts what you'd like to cover.", {
+										components: { b: <b /> },
+										comment: "This is a benefit listed on a 'Purchase a call with us' page",
+									} ) }
+								</span>
+							</li>
+						</ul>
+
+						<h4 className="concierge-support-session__sub-header">
+							{ translate(
+								'Reserve a 45-minute "Quick Start" appointment, and save %(saveAmount)s if you sign up today.',
+								{
+									args: {
+										saveAmount: formatCurrency( savings, currencyCode ),
+									},
+								}
+							) }
+						</h4>
+
+						<p>
+							<b>
+								{ translate(
+									'Book your call today for just {{del}}%(oldPrice)s{{/del}} %(price)s.',
+									{
+										components: { del: <del /> },
+										args: {
+											oldPrice: formatCurrency( fullCost, currencyCode ),
+											price: productDisplayCost,
+										},
+									}
+								) }
+							</b>{' '}
+							{ translate(
+								'Click the button below to confirm your purchase (sessions are currently limited to English language support).'
+							) }
+						</p>
+					</div>
+					<div className="concierge-support-session__column-doodle">
+						<img
+							className="concierge-support-session__doodle"
+							alt=""
```

Comment:
Let's fill in this alt text in the name of accessibility :)

Reply:
Fixed in [df60aa9](https://github.com/Automattic/wp-calypso/pull/33996/commits/df60aa99d4aeb47033d61ff3a92d8aecae22d942)